docs/content/1.documentation/2.headers/1.csp.md

```diff
@@ -215,15 +215,6 @@ export default defineNuxtConfig({
 - `"'nonce-{{nonce}}'"` placeholder: Include this value in any individual policy that you want to be governed by nonce. 
 
 
-::alert{type="warning"}
```

Comment:
I think this section should not be removed
The vite nonce is only inserted in dev mode, if I understand correctly

Reply:
I think all offending actions are only executed in dev mode as well. Styles should not be added through JavaScript by Nuxt in production as far as I know. They're just referenced as compiled CSS content in production, at least without changes to any configuration.
I have the fix contained in this PR running for a few of my sites for quite some time now and have not observed any issues (even with close monitoring done by Sentry).

Maybe @danielroe could quickly confirm or refute that style loading through JavaScript only happens in dev mode by vite and, by default, not by Nuxt in production?

Reply:
I'm pretty sure we had to use `unsafe-inline` in styles in our own doc website, because @nuxt/ui uses pinceau under the hood, and pinceau modifies styles at runtime.

Reply:
Ok, but that's somewhat 3rd party, no? The section talks about "Nuxt's mechanism for Client-Side hydration of styles" so maybe the wording should be changed then to reflect the actual reason for the recommendation better then if the recommendation in that form is even still needed.

Reply:
I'll unsubscribe from this PR to reduce noise, just tag me again if you need me